utils/file_manager.py

```
@@ -0,0 +1,128 @@
+        
+        print("Checking potential locations:")
+        for path in potential_paths:
+            print(f"  Checking: {path}")
+            if os.path.exists(path) and os.path.isfile(path):
+                abs_path = os.path.abspath(path)
+                found_path = abs_path
+                if system == "windows":
+                    found_path = found_path.replace("\\", "\\\\")                   
+                print(f"    ✓ FOUND: {abs_path}")
+            else:
+                print(f"    ✗ Not found")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)